apps/micro_react/src/app/app.tsx

```diff
@@ -6,16 +6,17 @@ import { Banner, RatingStar } from '@micro-frontend-react/common-ui';
 import { Characters } from '../components/characters';
 
 const App = () => {
-  const [showCharacters, setShowCharacters] = useState(false);
+  const [showCharacters, setShowCharacters] = useState<boolean>(false);
```

Comment:
this is not completely necessary but is a good thing to do. It's nice to specify the type of state through the concept of generics. It would be awesome if you could read a bit about generics. ;)